python/cudf/cudf/core/index.py

```diff
@@ -276,6 +276,20 @@ def argsort(self, ascending=True, **kwargs):
         indices = self._values.argsort(ascending=ascending, **kwargs)
         return cupy.asarray(indices)
 
+    @property
+    def empty(self):
+        """
+        Indicator whether Index is empty.
+
+        True if Index is entirely empty (no items).
+
+        Returns
+        -------
+        out : bool
+            If Index is empty, return True, if not return False.
+        """
+        return super().empty
```

Comment:
We shouldn't have to do this. By inheritance, `Index` will already have an `empty` method.

Reply:
Ditto for all other subclasses of `Frame`